app/gui2/src/components/CircularMenu.vue

```diff
@@ -76,26 +123,61 @@ const emit = defineEmits<{
   opacity: unset;
 }
 
-.override-output-context-button {
-  position: absolute;
-  left: 9px;
-  top: 8px;
+.inactive {
+  pointer-events: none;
+  opacity: 10%;
 }
 
 .output-context-overridden {
   opacity: 100%;
   color: red;
 }
 
-.docs-button {
+.slot1 {
```

Comment:
Please, add a comment how the slots are placed in the menu. Is Slot1 the bottom most, or leftmost? Are they numbered clockwise, or counterclockwise?